packages/create-plugin/templates/common/.config/webpack/webpack.config.ts

```diff
@@ -34,7 +35,9 @@ __webpack_public_path__ =
 `,
 });
 
-const config = async (env): Promise<Configuration> => {
+export type Env = Record<string, boolean | string | Env>
```

Comment:
Ah, this can't be `false`, if you do `--env test=false` you get `'false'` as a string

```suggestion
export type Env = Record<string, true | string | Env>
```